modules/user.paths.yaml

```diff
@@ -1314,6 +1314,52 @@ user-image:
         description: Success
       default:
         description: Unexpected error.
+user-push:
+  post:
+    tags:
+      - User
+    summary: add push notification token to user
+    description: Add/update a device/platform specific token for push notifications
+    consumes:
+      - application/json
+    parameters:
+      - name: userToken
+        in: header
+        type: string
+      - name: body
+        in: body
+        schema:
+          $ref: 'https://skedgo.github.io/tripgo-api/modules/user.definitions.yaml#/json_push_token'
+        description: push notification data
```

Comment:
Description does not match what's in that schema. The schema is just about tokens, not about data of a push notification itself.